src/modules/assertions/require_inline_coding.ts

```diff
@@ -0,0 +1,12 @@
+import { context } from "@actions/github";
+import _ from "lodash";
+
+export const  requireInlineCoding = () => {
+  const sol_file_Regex = /http:\/\/\S+.sol/g;
+  if (sol_file_Regex.test(JSON.stringify(context.payload?.pull_request?.body))) {
```

Comment:
Do we know what conditions lead to `payload` being null/undefined?  The `?` here suggest that it is possible, but we should better understand and handle that case.